addons/website/static/src/scss/website.scss

```diff
@@ -1433,6 +1433,14 @@ $ribbon-padding: 100px;
     :not(.o_cookies_popup) {
         bottom: 0;
     }
+    .modal-dialog {
+        margin: 0
+    }
+    .container {
+        @media (max-width: 991px) {
+            padding-bottom: 15px;
```

Comment:
if I understand this only adds a margin to the bottom. if we want that it would make more sense in my opinion to handle it like other margin, with class like pb16 on that:
https://github.com/odoo/odoo/blob/15.0/addons/website/views/website_templates.xml#L1848

Reply:
Yep, this part is adding a padding to the bottom to every small/mid screens. In this case `pb16` would add a padding on large screens too, which we don't want here